contributor-docs/adding-template.md

```diff
@@ -1,85 +1,180 @@
 
-There are also some disadvantages to running the sandboxes in a browser. The main one is that we lose flexibility: it's not possible to run CLI commands and in some cases we don't support custom configuration (like a custom webpack configuration). That's why we developed a new kind of sandbox called Container in September.
+There are also some disadvantages to this approach. When sandboxes run in the browser, we lose flexibility. It's no longer possible to run Command Line Interface (CLI) commans and in some cases custom configations are not supported, for example a custom webpack configuration. This is why we have developed a new kind of sandbox called **Container**, which we released in September 2018.
 
 ### Containers
 
-We, unlike sandboxes, execute Container sandboxes on a server. We can run more end-to-end projects with container sandboxes, like `Next.js`. We allow you to run any command on a container, everything that works locally works in a container. Containers are also used to build bigger projects.
+Unlike sandboxes, **containers** are executed on a server. This makes it possible to create projects that are end-to-end based, like `Next.js` with CodeSandbox, and also makes it possible to build bigger projects. **Containers** let you run any command, and everything that works locally will also work in container.
 
-There are some limitations to this approach though: we only allow signed in users to work on a container and the amount of container sandboxes a user can have is limited (this limit changes as we make containers more stable). You also can't edit containers while offline and it's not possible to edit containers when viewing from an embed.
+However, like the sandboxes, **containers** also come with some limitations. In order to work on a container, you need to be signed in as a user, you can't edit containers while offline, it's not possible viewing them from an embed, and ~~you can only have a limited amount of container based projects~~ ([containers are now out of beta](https://medium.com/@compuives/codesandbox-containers-out-of-beta-improvements-be79679df4cc)).
 
 ### Which one to choose?
 
-The template defines whether a sandbox is executed in a container or in the browser. This means that it's your choice where to execute the sandbox.
+As you have read above, the template type determines whether a project is executed in a *sandbox in a browser* or in a *container on a server*. This means that the template type you have to choose, depends on your specific use-case and where you want your project to be executed.
+
+If you would like to demonstrate CLI functionality, we recommend that you use a **container** type template, and if you want to demonstrate a JavaScript framework (like `create-react-app`) we recommended using a**sandbox** type template.
```

Comment:
```suggestion
If you would like to demonstrate CLI functionality, we recommend that you use a **container** type template, and if you want to demonstrate a JavaScript framework (like `react`) we recommended using a**sandbox** type template.
```

Create React App is the CLI, maybe makes sense for this to be react

Reply:
`React`, capital "r". 🙂